src/Tizen.NUI/src/public/TransitionAnimations/TransitionAnimations.cs

```diff
@@ -3,60 +3,139 @@
 using System.ComponentModel;
 using System.Text;
 using Tizen.NUI;
+using Tizen.NUI.BaseComponents;
 
 namespace Tizen.NUI
 {
+    /// <summary>
+    /// Data of animation data for transition
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class TransitionAnimationData
+    {
+        /// <summary>
+        /// start time of animation
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int StartTime { get; set; }
+
+        /// <summary>
+        /// end time of animation
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int EndTime { get; set; }
+
+        /// <summary>
+        /// property of animation
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string Property { get; set; }
+
+        /// <summary>
+        /// destination value of animation
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string DestValue { get; set; }
```

Comment:
from next time, please do not use short word.
"DestValue" should be "DestinationValue"

Reply:
Done.